src/Aquifer.API/Endpoints/Resources/Content/AssignEditor/Endpoint.cs

```diff
@@ -142,22 +142,21 @@ private static void SetDraftVersionStatus(bool isTakingBackFromReviewPending,
         UserEntity userToAssign,
         ResourceContentVersionEntity draftVersion)
     {
-        if (userToAssign.Role is not UserRole.Manager || !Constants.ManagerReviewStatuses.Contains(originalStatus))
+        if (userToAssign.Role is not UserRole.Manager && !Constants.CompanyReviewStatuses.Contains(originalStatus))
```

Comment:
Was this `||` to `&&` switch intentional?

Reply:
Yes, otherwise, when a item in `CompanyReview` status is kicked back to an editor, it would revert the status to `EditorReview` as well which is not what should happen according to this ticket